api/services/ocean/commons/parameters/deleteNodes.yaml

```diff
@@ -1,4 +1,4 @@
-in: path
+in: query
```

Comment:
No need to change, it was valid update

Reply:
http://help.dev.spotinst.com/spotinst-api/ocean/ocean-cloud-api/ocean-for-aws/launch-specifications/delete/